float64 lat
float64 longitude
float64 alt